source/leuven/slurm-accounting.rst

```diff
@@ -3,58 +3,81 @@
-the technical aspects of accounting are explained.
+The :ref:`wICE <wice_t2_leuven>` cluster uses the Slurm Account Manager (SAM) 
+for credit accounting.
+The reason is that having a valid Slurm account with sufficient credits in it
+is mandatory for running jobs on KU Leuven Tier-2 machines. 
+We developed so called SAM-commands to mimic the behavior of previously used 
+MAM-commands (from the MOAB account manager). 
+From the user point of view it should mostly be enough to change the prefix ``mam`` 
+in the accounting command into ``sam``.
+
+The accounting system is very similar to a regular bank. 
+Individual users have accounts that will be charged for the jobs they run. 
+However, the number of credits on such accounts is fairly small, so research projects 
+will typically have one or more project accounts associated with them. 
+Users that are project members can have their project-related jobs charged to such a 
+project account. 
+In this how-to, the technical aspects of accounting are explained.
 
 Checking an account balance
 ---------------------------
 
-Since no calculations can be done without credits, it is quite useful to determine the amount of credits at your disposal. This can be done by executing the following command on any KU Leuven Tier-2 node::
+Since no calculations can be done without credits, it is quite useful to determine the 
+amount of credits at your disposal. 
+This can be done by executing the following command on any KU Leuven Tier-2 node::
 
    $ sam-balance
 
-This will provide an overview of the balance on the user's personal account, as well as on all project accounts the user has access to.
+This will provide an overview of the balance on the user's personal account, as well as 
+on all project accounts the user has access to.
 
 
 Running jobs: accounting workflow
 ---------------------------------
 
-When a job is submitted using ``sbatch`` or ``srun`` and it has to be charged against a project account, the name of the project has to be specified as an option. In case of the introduction
-credits the project account should be specified as  ``default_project``::
+When a job is submitted using ``sbatch`` or ``srun``, it has to be charged against a 
+project account.
+The name of the project has to be specified as an option. 
+If a user requests introductory credits, the assigned project name carries the VSC-id
+of the user, e.g. ``intro_vsc3....``, where the dots ``.`` signify the user's uniqe VSC-id.
 
    $ sbatch -A lp_my_project run-job.slurm
+   or
+   $ sbatch -A intro_39999 run-job.slurm
 
-If the account to be charged, i.e., ``lp_my_project``, has insufficient credits for the job, the user receives a warning at this point.
+If the account to be charged, i.e., ``lp_my_project``, has insufficient credits for the 
+job, the user receives a warning at this point, and the job will never enter the queue.
```

Comment:
```suggestion
job, the user receives a warning at this point, and the job will not enter the queue.
```
("never" suggests that there could be multiple future times at which the job may still somehow be eligible for entering the queue, whereas in reality it is just a "one-off" thing: either the job enters the queue when it is submitted, or it gets rejected)

Reply:
Alright; it reads like this now:
"If the account to be charged, i.e., ``lp_my_project``, has insufficient credits for the 
job, the user receives a warning at this point, and the job will start until the account
is topped up with sufficient credits."